runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -1374,8 +1374,7 @@ TR::Register *J9::X86::TreeEvaluator::multianewArrayEvaluator(TR::Node *node, TR
       }
 
    // temp2Reg = firstDimLenReg * 16 (discontiguousArrayHeaderSizeInBytes)
-   generateRegRegInstruction(MOVRegReg(),  node, temp2Reg, firstDimLenReg, cg);
-   generateRegImmInstruction(SHLRegImm1(), node, temp2Reg, 4, cg);
+   generateRegRegImmInstruction(IMulRegRegImms(4), node, temp2Reg, firstDimLenReg, zeroArraySize, cg);
```

Comment:
@dchopra001  This is the update you suggested me, please let me know if this change will be part of https://github.com/eclipse/openj9/pull/11320 so I can remove it from here

Reply:
The change will be part of the above mentioned PR (and I've ported an equivalent fix to Z as well). So you can remove it from here. Thanks!